sections/gov/components/Proposal/Results.tsx

```diff
@@ -30,12 +30,14 @@ const Results: React.FC<ResultsProps> = ({ hash }) => {
 				const currentElectionMembers = CouncilNominations as any;
 				const mappedProfiles = [] as any;
 
-				currentElectionMembers[electionCount].forEach((member: any) => {
-					mappedProfiles.push({
-						address: member.address,
-						name: member.discord,
+				if (currentElectionMembers[hash]) {
```

Comment:
seems like the react hook `useEffect` is complaining that its missing `hash` (it shows on GitHub)

Reply:
thanks @evgenyboxer